papaya-core/src/main/java/pl/edu/agh/papaya/service/jira/JiraAuthorisationService.java

```diff
@@ -0,0 +1,82 @@
+package pl.edu.agh.papaya.service.jira;
+
+import com.google.api.client.auth.oauth.OAuthParameters;
+import com.google.api.client.http.GenericUrl;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpRequestFactory;
+import com.google.api.client.http.HttpResponse;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.security.NoSuchAlgorithmException;
+import java.security.spec.InvalidKeySpecException;
+import java.util.Optional;
+import java.util.Scanner;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.json.JSONObject;
+import org.springframework.stereotype.Service;
+import pl.edu.agh.papaya.service.project.ProjectService;
+
+@Service
+@Getter
+@Setter
+@NoArgsConstructor
+@SuppressWarnings("SummaryJavadoc")
+public class JiraAuthorisationService {
+
+    //todo: Passing via @Value
```

Comment:
Ja wiem, że Travis by się wywalił na komentarzu `//TODO` wielkimi literami, ale to chodzi o to, żeby nie zapomnieć :stuck_out_tongue: Już któraś osoba tak robi, więc chyba będę musiała poszukać tego regexa

Reply:
Dodano przesyłanie wartości